hmf-common/src/main/java/com/hartwig/hmftools/common/variant/impact/VariantTranscriptImpact.java

```diff
@@ -79,6 +82,12 @@ public static List<VariantTranscriptImpact> fromVariantContext(final VariantCont
     public static VariantTranscriptImpact fromVcfData(final String data)
     {
         String[] items = data.split("\\" + VAR_TRANS_IMPACT_ITEM_DELIM, -1);
+
+        // dirty fix: append items if it is less than 7 to prevent out of bound exceptions
```

Comment:
Some tests (for example those in orange) have entires where this array has fewer entries than 7. In those cases, the tests would fail with an index out of bounds exception.

I lack the domain knowledge to determine if this is a sign of a more serious problem with the data, or if this is expected sometimes. For now I implemented this fix, but maybe it is worth looking into how to handle these situations.